src/kaleidoscope/hardware/kbdfans/KBD4x.h

```diff
@@ -19,37 +19,36 @@
 
 #ifdef ARDUINO_AVR_KBD4X
 
+#define KALEIDOSCOPE_BOOTLOADER_FLIP 1
```

Comment:
I mostly understand why we need to do this, but I feel like the definition should have "WORKAROUND" or something in the name.